src/boot/bootctl-uki.c

```diff
@@ -0,0 +1,96 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "bootctl.h"
+#include "bootctl-uki.h"
+#include "fd-util.h"
+#include "parse-util.h"
+#include "pe-header.h"
+
+#define DOS_FILE_MAGIC "MZ"
+#define PE_FILE_MAGIC  "PE\0\0"
```

Comment:
given this is a binary magic, maybe also define these as `static const uint8_t[]`, like the section names below? would have the benefit you could size them exactly, no trailing NUL byte would be appended.